NeoML/src/Dnn/Layers/AccuracyLayer.cpp

```diff
@@ -50,7 +50,8 @@ void CAccuracyLayer::Reshape()
 		&& inputDescs[0].Depth() == 1 );
 
 	outputDescs[0] = CBlobDesc( CT_Float );
-
+	CheckArchitecture( inputDescs[0].ObjectSize() == inputDescs[1].ObjectSize()
+		|| inputDescs[1].ObjectSize() == 1, GetName(), "Object size mismatch between inputs" );
```

Comment:
Wrong error description. As an alternative I suggest "Second input object size must match or be equal to 1" (or anything that covers both cases)